src/signals/incident/components/IncidentNavigation/IncidentNavigation.tsx

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MPL-2.0
-// Copyright (C) 2018 - 2022 Gemeente Amsterdam
+// Copyright (C) 2018 - 2023 Gemeente Amsterdam
```

Comment:
Did you change anything here?

Reply:
I had, but then overwritten it again except, it seems, for the date change.